docs/section11/Rule11-8.md

```diff
@@ -0,0 +1,65 @@
+    - otherwise, rule is not applicable for this swh_bat: `else: NOT_APPLICABLE`
+
+        ## Rule Logic: 
+    - use get_SWH_equipment_associated_with_each_SWH_bat to get the SWH use types and SWH use in the building: `swh_bats_and_equip_dict = get_SWH_equipment_associated_with_each_SWH_bat(B_RMD)`
+    - get a dictionary of all SWH equipment connected to the current SWH bat in the baseline model: `swh_bats_and_equip_dict_this_use = swh_bats_and_equip_dict[swh_bat]`
+    - get the quantity of service water heating distribution systems serving this building area type: `num_swh_systems = len(swh_bats_and_equip_dict_this_use["SWHDistribution"])`
+    - get the quantity of service water heating equipment serving this building area type: `num_swh_equipment_this_use = len(swh_bats_and_equip_dict_this_use["SWHHeatingEq"])`
+    - create a boolean is_referenced_in_other_bats and set it to false: `is_referenced_in_other_bats = false`
+    
+    - if there is exactly one system, check that the system is not used in any of the other BATs: `if num_swh_systems == 1:`
+        - create a variable swh_dist_id and set it to the id of the swh distribution system for this bat: `swh_dist_id = swh_bats_and_equip_dict_this_use["SWHDistribution"][0]`
+        - loop through each bat: `for other_swh_bat in swh_bats_and_equip_dict:`
+        - if other_swh_bat is not equal to the current swh_bat: `if other_swh_bat != swh_bat:`
+            - check whether the swh_dist_id is in the list of SWH Distribution system for the other SWH_bat: `if swh_dist_id in? swh_bats_and_equip_dict[other_swh_bat]["SWHDistribution"]:`
+            - set the boolean is_referenced_in_other_bats to true: `is_referenced_in_other_bats = true`
+    - create a boolean called multiple_segments_with_bat_other and set it to false.  This boolean will tell us if the a) the swh_bat is "OTHER"; b) there are multiple building segments with swh_bat equal to "OTHER": `multiple_segments_with_bat_other = false` 
+    - check whether the swh_bat is "OTHER": `if swh_bat == "OTHER":`
+        - now create a variable num_segments and set it to 0.  This variable will be used to count the number of building segments with swh_bat == "OTHER": `num_segments = 0`
+        - now look at each building segment in the building: `for building_segment in B_RMD:`
+            - get the building segment swh_bat.  If it's "OTHER" increment, num_segments: `if (get_building_segment_SWH_bat(B_RMD, building_segment) == "OTHER"): num_segments += 1`
+        - check whether there are multiple building segments with swh_bat "OTHER", if so, set the boolean other equal to true: `if num_segments > 1: multiple_segments_with_bat_other = true`
+
+    ## Rule Assertion: 
+    - Case1: there is more than one swh system in the building segment: FAIL: `if num_swh_systems > 1: FAIL`
```

Comment:
I think this shoudl be an or I think this should also check that num_swh_equipment_this_use > 1, right? Or am I missing something.